omnipaxos_ui/src/app.rs

```diff
@@ -73,10 +72,11 @@ impl App {
         peers.sort();
         let nodes: Vec<Node> = peers
             .into_iter()
-            .map(|pid| {
+            .enumerate()
```

Comment:
sort the pids first so that the colors are the same regardless of which node/dashboard we start with.

Reply:
It sorted at the line `72` above